client/src/Components/Card/BusinessTotalCard.js

```diff
@@ -1,25 +1,16 @@
-import React, { Component } from "react";
```

Comment:
is this meant to be commented out?

Reply:
Yes. 
A BusinessTotalCard does not need anymore. That could be inside BusinessRatioCard.
Trying to clean up unnecessary cards.